src/pretix/api/serializers/item.py

```diff
@@ -364,6 +385,7 @@ def create(self, validated_data):
         variations_data = validated_data.pop('variations') if 'variations' in validated_data else {}
         addons_data = validated_data.pop('addons') if 'addons' in validated_data else {}
         bundles_data = validated_data.pop('bundles') if 'bundles' in validated_data else {}
+        program_times_data = validated_data.pop('program_times') if 'program_times' in validated_data else {}
```

Comment:
This requires extension of API docs and tests, also am I reading this correctly that this allows creating program times through the API, but neither updating nor reading them is possible?